modoboa/locale/de/LC_MESSAGES/django.po

```diff
@@ -426,23 +420,25 @@ msgstr "Passwort"
 msgid "last login"
 msgstr "Letzte Anmeldung"
 
+#: core/models.py:70
+msgid "language"
+msgstr "Sprache"
```

Comment:
Looks like you did not update the source file. Have you run the following command:
```python
> django-admin makemessages -a
```
?